build_tools/python/e2e_test_framework/definitions/iree_definitions.py

```diff
@@ -98,55 +98,90 @@ class ModuleExecutionConfig(object):
   extra_flags: List[str] = dataclasses.field(default_factory=list)
 
 
-@dataclass(frozen=True)
-class _MLIRDialectPair(object):
-  """MLIR dialect with its unique artificial id."""
-  # Unique artificial id.
-  id: int
-  # Name of an IREE supported input type (--iree-input-type).
-  dialect_name: str
-
-
-# Please use and update the next id when adding a new dialect, so we won't reuse
-# the old deprecated id.
-# Next ID: 4
-class MLIRDialectType(_MLIRDialectPair, Enum):
+# Value should be the name of an IREE supported input type (--iree-input-type).
+class MLIRDialectType(Enum):
```

Comment:
Why was this change to the enum reverted?

Reply:
MLIRDialectType can now be part of the ImportConfig, so no need to have a dedicated id for it.

Basically if we are going to output different dialect, we just create a new ImportConfig for it.